### PR TITLE
[OCPBUGS-18394] Multiarch compute IBM Z - Add extra step to procedure

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OCPBUGS-18394

Link to docs preview:
https://66472--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm

QE review: 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/multi-architecture-verifying-cluster-compatibility.adoc [hunk 1]
```diff
@@ -44,13 +44,16 @@ endif::ibm-power[]
 $ oc adm release info -o json | jq .metadata.metadata
```
Comment (aleskandro) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:44]:
```suggestion
$ oc adm release info -o jsonpath="{ .metadata.metadata}"
```

File: modules/machine-user-infra-machines-ibm-z-kvm.adoc [hunk 2]
```diff
@@ -16,6 +16,31 @@ You can create more {op-system-first} compute machines for your cluster by using
 * An HTTP or HTTPS server is set up.
 
 .Procedure
+// Step 1 is a workaround for https://issues.redhat.com/browse/OCPBUGS-18394
+// Can be removed when bug is fixed.
+. Disable UDP aggregation
```
Comment (mburke5678) [inline modules/machine-user-infra-machines-ibm-z-kvm.adoc:21]:
```suggestion
. Disable UDP aggregation.
```

File: modules/machine-user-infra-machines-ibm-z-kvm.adoc [hunk 3]
```diff
@@ -16,6 +16,31 @@ You can create more {op-system-first} compute machines for your cluster by using
...
++
+Currently, UDP aggregation is not supported on {ibmzProductName} and is not automatically deactivated on multi-architecture compute clusters with an `x86_64` control plane with additional `s390x` compute machines. To ensure that the addtional compute nodes are added to the cluster correctly, you must manually disable UDP aggregation.
```
Comment (mburke5678) [inline modules/machine-user-infra-machines-ibm-z-kvm.adoc:23]: To clarify what has the additional machines. Probably not needed for experts, but might help translators.
```suggestion
Currently, UDP aggregation is not supported on {ibmzProductName} and is not automatically deactivated on multi-architecture compute clusters with an `x86_64` control plane and additional `s390x` compute machines. To ensure that the addtional compute nodes are added to the cluster correctly, you must manually disable UDP aggregation.
```

**Comment by mburke5678** [comment]
@SNiemann15 A few nits. Oherwise lgtm. However, I wonder if this might be better as a prerequisite, as it isn't truly a step to install a compute machine.

**Comment by holgwolf** [comment]
lgtm

**Review by opayne1 — APPROVED** [review]

**Pull request merged**